src/js/script.js

```diff
@@ -119,6 +119,9 @@ jQuery(function($) {
 				var code = $(this);
 				var lines = code.html().split(/\n/).length;
 				var numbers = [];
+				if (lines > 1) {
```

Comment:
This just gives me an extra unwanted line: <img width="673" alt="Bildschirmfoto 2019-10-17 um 08 27 41" src="https://user-images.githubusercontent.com/888679/67027048-c10f4780-f0bd-11e9-9bf5-661a6f3bb9f1.png">